### PR TITLE
bump open-aea to v1.15.0

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

This PR bumps open-aea to v1.15.0

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # 0.1.5 (2022-07-29)
 
 Autonomy:
-- Depends on `open-aea>=1.14.0.post1`
+- Depends on `open-aea>=1.15.0`
```
Comment (DavidMinarsch) [inline HISTORY.md:6]: Wrong

File: tox.ini [hunk 2]
```diff
@@ -362,7 +362,7 @@ deps =
     black==21.6b0
     click==8.0.2
     isort==5.9.3
-commands = aea generate-all-protocols --no-bump --check-clean
+commands = aea generate-all-protocols --check-clean
```
Comment (angrybayblade) [inline tox.ini:365]: `--no-bump` was causing the command to fail, I guess this was caused by https://github.com/valory-xyz/open-aea/pull/243

Comment (DavidMinarsch) [reply tox.ini:365]: Should be reverted after merging?

File: packages/valory/protocols/http/__init__.py [hunk 3]
```diff
@@ -20,7 +20,7 @@
 """
 This module contains the support resources for the http protocol.
 
-It was created with protocol buffer compiler version `libprotoc 3.19.4` and aea version `1.13.0`.
```
Comment (DavidMinarsch) [inline packages/valory/protocols/http/__init__.py:23]: Weird that it still was on 1.13.0 and not on 1.14...

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**